src/components/TabItem/TabItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types';
+
+type Props = {
+  tabItem: Tab;
+  selectedTab: Tab;
+  onTabSelect: (tab: Tab) => void;
```

Comment:
Check `isActive` outside of tab component
```suggestion
  tabItem: Tab;
  isActive: Boolean;
  onTabSelect: (tab: Tab) => void;
```